src/plugins/data-card-tool/data-card-tool.scss

```diff
@@ -218,45 +201,47 @@ $default-button-border-color: #979797;
 
       input {
         width: 100%;
-        height: 100%;
+        height: 97%;
```

Comment:
I experimented with this and a simple `100%` seems to work as well.

Reply:
>The reason I used 97% is that at 100% height the input just slightly covers up the borders, making the borders on the value side thinner than the borders on the label side.

The reason that something like `97%` is almost certainly not what you want is that the effect you're trying to achieve is a pixel-based effect, namely to prevent a border overlap of one pixel. While in this particular case `97%` happens to have the desired effect, if the design changes so that the parent element changes its height, then this value has to change to `95%` or `98%`, so there's a co-dependency between the two values. Given that the goal is to prevent a border overlap of one pixel, `calc(100% - 1px)` accomplishes that directly without the co-dependency.

```suggestion
        height: calc(100% - 1px);
```